arxiv/mail/templates/head.html

```diff
@@ -0,0 +1,81 @@
+<meta name="viewport" content="width=device-width">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>{% block email_title %}A message from arXiv{% endblock %}</title>
+<!--[if mso]>
+    <style>
+        * {
+            font-family: sans-serif !important;
+        }
+    </style>
+<![endif]-->
+
+<!--[if !mso]>
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,700' rel='stylesheet' type='text/css'>
```

Comment:
I'm a little unsure about including refs to third-party resources in our e-mails, since it could be seen as an opportunity for tracking

Reply:
I'm okay with leaving it out if you don't feel great about it. Many companies and templates do include webfonts, but many also don't. If we don't: we will be at the whims (a bit) of system font defaults. However, if we do: some email clients still don't support webfonts anyway (Gmail started supporting them just this year and that's almost 30% of average viewerships), so I have some decent fallbacks in place.

Basically not a big deal, I'm happy to go with your call on it.

Reply:
I'm against facilitating tracking of arXiv users by third parties.

Reply:
> I'm okay with leaving it out if you don't feel great about it. Many companies and templates do include webfonts, but many also don't. If we don't: we will be at the whims (a bit) of system font defaults. However, if we do: some email clients still don't support webfonts anyway (Gmail started supporting them just this year and that's almost 30% of average viewerships), so I have some decent fallbacks in place.
> 
> Basically not a big deal, I'm happy to go with your call on it.
> (I mean yes, the typographer in me sobs about using Arial, but it's all good.)

It's not so much about including fonts as it is about loading the fonts from someone else. So if that font is something that we want to use, we could consider serving it from our own CDN.

Reply:
We certainly could host the fonts ourselves, and this is an issue that should be raised outside of the context of just email.

Note that we are already using third party APIs/CDNs on our web pages (not email, true, but odd that this hasn't come up before):
https://github.com/arXiv/arxiv-base/blob/d68bca8329a6697e34ea508cee76262477b7aa76/arxiv/base/static/sass/arxivstyle.sass#L1
https://github.com/arXiv/arxiv-base/blob/d68bca8329a6697e34ea508cee76262477b7aa76/arxiv/base/templates/base/head.html#L15

Reply:
You're absolutely right about having done this before, and we should look at hosting these ourselves as a task in itself. Thanks to @bdc34 for helping to keep us on the straight and narrow